homeassistant/components/hardware/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""The Hardware integration."""
+from __future__ import annotations
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.integration_platform import (
+    async_process_integration_platforms,
+)
+from homeassistant.helpers.typing import ConfigType
+
+from . import websocket_api
+from .const import DOMAIN
+from .models import HardwareProtocol
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up Hardware."""
+    hass.data[DOMAIN] = {"hardware_platform": {}}
+
+    websocket_api.async_setup(hass)
+    await async_process_integration_platforms(hass, DOMAIN, _register_hardware_platform)
```

Comment:
only have to call this on first incoming WS command to avoid doing I/O on startup.

Reply:
That means we won't install integrations associated with the board (`zha`, `rpi_power` etc.) until the user visits the hardware screen in frontend?